### DISCUSSION TITLE
can’t rename comics in-app, can”t report bug in-app

### DISCUSSION THREAD

**Discussion by deatomize** [community/community#36929] [Other Feature Feedback, Questions, & Ideas]
i bought this app a few months ago and thought it was perfect. however, i’ve been having issues.

i’m unable to rename my comics because it says i can’t change the extension, when i’m not even doing so. also, i can’t use the “send feedback/report bugs” button because it doesn’t even do anything. i’ve tried so many times.

finally, i tried looking at your website for a way to contact you to say this, but your website is offline. that caused me to end up here reporting it.

i pray you see this, tim! if i had a way to contact support from you guys, it would have saved me from typing all of this lol.

my phone is the iPhone SE 2020, if that matters any.

**Comment by radar-blip** [comment]
I think you may be in the wrong place.
If you are looking for an App here on GitHub, you can try searching for the name of the app.
If it is here somewhere (on GitHub) maybe you will be able to contact Tim.

**Comment by Mobsapps** [comment]
The process to rename comics on the app is simple. You can either tap the pencil icon or select the three dots in the top right corner of the comic. Then, you will see a list of options. From there, tap "rename" and type in your new title for this comic.

This is a great way to add some personalization to your favorite comics and make them more **[memorable!](<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/SULMAN~1/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/SULMAN~1/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<div data-ui-name=Flex name=phrase style='border-top-style:initial;border-right-style:
initial;border-left-style:initial;border-top-color:initial;border-right-color:
initial;border-left-color:initial;border-image: initial;font-variant-ligatures: normal;
font-variant-caps: normal;font-variant-numeric: inherit;font-variant-east-asian: inherit;
font-stretch: inherit;vertical-align:initial;display:flex;flex: 1 1 auto;
box-sizing: border-box;min-height: 45px;position:sticky;orphans: 2;text-align:
start;widows: 2;-webkit-text-stroke-width: 0px;text-decoration-thickness: initial;
text-decoration-style: initial;text-decoration-color: initial;width:var(--phrase_width);
left:var(--checkbox_width)'>

<div style='vertical-align:initial;margin-right:0px;padding-bottom:0px;
padding-top:0px'>

<div data-ui-name=Flex style='vertical-align:initial;display:flex;margin-right:
0px;padding-bottom:0px;padding-top:0px'>

<div style='vertical-align:initial;white-space:break-spaces'><span data-ui-name=Link.Text
style='border-top-style:initial;border-right-style:initial;border-left-style:
initial;border-color:transparent;border-image: initial;vertical-align:initial;
transition: border-bottom-color 0.15s ease-in-out 0s'>

<div data-ui-name=Flex style='vertical-align:initial;display:inline-flex'>

<div data-ui-name=Flex style='vertical-align:initial;display:inline-flex'>

<div style='vertical-align:initial;margin-right:0px;padding-bottom:0px;
padding-top:0px'></div>

</div>

</div>

</div>

</div>

</div>

</div>

</div>

<div data-ui-name=Flex name=intents style='border-top-style:initial;border-right-style:
initial;border-left-style:initial;border-top-color:initial;border-right-color:
initial;border-left-color:initial;border-image: initial;font-variant-ligatures: normal;
font-variant-caps: normal;font-variant-numeric: inherit;font-variant-east-asian: inherit;
font-stretch: inherit;vertical-align:initial;display:flex;flex: 1 1 auto;
box-sizing: border-box;min-height: 45px;orphans: 2;text-align:start;widows: 2;
-webkit-text-stroke-width: 0px;text-decoration-thickness: initial;text-decoration-style: initial;
text-decoration-color: initial;width:var(--intents_width);justify-content: center'>

<div style='vertical-align:initial;margin-right:0px;padding-bottom:0px;
padding-top:0px'>

<div data-ui-name=Flex style='font-style:inherit;font-variant:inherit;
font-weight:inherit;font-stretch: inherit;vertical-align:initial;display:flex;
align-items: flex-start'>

<div style='vertical-align:initial;margin-right:0px;padding-bottom:0px;
padding-top:0px'>

<div data-ui-name=Flex style='vertical-align:initial;display:flex'>

<div data-ui-name=Tooltip aria-describedby=ui-kit-8608 style='vertical-align:
initial;display:flex;margin-right:0px;padding-bottom:0px;padding-top:0px'>

<div data-ui-name=Flex data-at=intent-badge-informational style='vertical-align:
initial;display:flex;margin-right:0px;padding-bottom:0px;padding-top:0px'>

<div data-ui-name=Flex style='vertical-align:initial;display:flex;border-radius: 4px;
cursor:pointer !important;align-items: center;justify-content: center'></div>

</div>

</div>

</div>

</div>

</div>

</div>

</div>

</span>

<div data-ui-name=Box data-at=ko-icon style='vertical-align:initial;display:
inline-block'>

<div data-ui-name=Flex name=phrase style='border-top-style:initial;border-right-style:
initial;border-left-style:initial;border-top-color:initial;border-right-color:
initial;border-left-color:initial;border-image: initial;font-variant-ligatures: normal;
font-variant-caps: normal;font-variant-numeric: inherit;font-variant-east-asian: inherit;
font-stretch: inherit;vertical-align:initial;display:flex;flex: 1 1 auto;
box-sizing: border-box;min-height: 45px;position:sticky;orphans: 2;text-align:
start;widows: 2;-webkit-text-stroke-width: 0px;text-decoration-thickness: initial;
text-decoration-style: initial;text-decoration-color: initial;width:var(--phrase_width);
left:var(--checkbox_width)'>

<div style='vertical-align:initial;margin-right:0px;padding-bottom:0px;
padding-top:0px'>

<div data-ui-name=Flex style='vertical-align:initial;display:flex;margin-right:
0px;padding-bottom:0px;padding-top:0px'>

<div style='vertical-align:initial;white-space:break-spaces'><span data-ui-name=Link.Text
style='border-top-style:initial;border-right-style:initial;border-left-style:
initial;border-color:transparent;border-image: initial;vertical-align:initial;
transition: border-bottom-color 0.15s ease-in-out 0s'>

<div data-ui-name=Flex style='vertical-align:initial;display:inline-flex'>

<div data-ui-name=Flex style='vertical-align:initial;display:inline-flex'>

<div style='vertical-align:initial;margin-right:0px;padding-bottom:0px;
padding-top:0px'></div>

</div>

</div>

</div>

</div>

</div>

</div>

</div>

<div data-ui-name=Flex name=intents style='border-top-style:initial;border-right-style:
initial;border-left-style:initial;border-top-color:initial;border-right-color:
initial;border-left-color:initial;border-image: initial;font-variant-ligatures: normal;
font-variant-caps: normal;font-variant-numeric: inherit;font-variant-east-asian: inherit;
font-stretch: inherit;vertical-align:initial;display:flex;flex: 1 1 auto;
box-sizing: border-box;min-height: 45px;orphans: 2;text-align:start;widows: 2;
-webkit-text-stroke-width: 0px;text-decoration-thickness: initial;text-decoration-style: initial;
text-decoration-color: initial;width:var(--intents_width);justify-content: center'>

<div style='vertical-align:initial;margin-right:0px;padding-bottom:0px;
padding-top:0px'>

<div data-ui-name=Flex style='font-style:inherit;font-variant:inherit;
font-weight:inherit;font-stretch: inherit;vertical-align:initial;display:flex;
align-items: flex-start'>

<div style='vertical-align:initial;margin-right:0px;padding-bottom:0px;
padding-top:0px'>

<div data-ui-name=Flex style='vertical-align:initial;display:flex'>

<div data-ui-name=Tooltip aria-describedby=ui-kit-1517 style='vertical-align:
initial;display:flex;margin-right:0px;padding-bottom:0px;padding-top:0px'>

<div data-ui-name=Flex data-at=intent-badge-informational style='vertical-align:
initial;display:flex;margin-right:0px;padding-bottom:0px;padding-top:0px'>

<div data-ui-name=Flex style='vertical-align:initial;display:flex;border-radius: 4px;
cursor:pointer !important;align-items: center;justify-content: center'></div>

</div>

</div>

</div>

</div>

</div>

</div>

</div>

</span>

<div data-ui-name=Box data-at=ko-icon style='vertical-align:initial;display:
inline-block'>

<div data-ui-name=Flex name=phrase style='border-top-style:initial;border-right-style:
initial;border-left-style:initial;border-top-color:initial;border-right-color:
initial;border-left-color:initial;border-image: initial;font-variant-ligatures: normal;
font-variant-caps: normal;font-variant-numeric: inherit;font-variant-east-asian: inherit;
font-stretch: inherit;vertical-align:initial;display:flex;flex: 1 1 auto;
box-sizing: border-box;min-height: 45px;position:sticky;orphans: 2;text-align:
start;widows: 2;-webkit-text-stroke-width: 0px;text-decoration-thickness: initial;
text-decoration-style: initial;text-decoration-color: initial;width:var(--phrase_width);
left:var(--checkbox_width)'>

<div style='vertical-align:initial;margin-right:0px;padding-bottom:0px;
padding-top:0px'>

<div data-ui-name=Flex style='vertical-align:initial;display:flex;margin-right:
0px;padding-bottom:0px;padding-top:0px'>

<div style='vertical-align:initial;white-space:break-spaces'><span data-ui-name=Link.Text
style='border-top-style:initial;border-right-style:initial;border-left-style:
initial;border-color:transparent;border-image: initial;vertical-align:initial;
transition: border-bottom-color 0.15s ease-in-out 0s'>

<div data-ui-name=Flex style='vertical-align:initial;display:inline-flex'>

<div data-ui-name=Flex style='vertical-align:initial;display:inline-flex'>

<div style='vertical-align:initial;margin-right:0px;padding-bottom:0px;
padding-top:0px'></div>

</div>

</div>

</div>

</div>

</div>

</div>

</div>

<div data-ui-name=Flex name=intents style='border-top-style:initial;border-right-style:
initial;border-left-style:initial;border-top-color:initial;border-right-color:
initial;border-left-color:initial;border-image: initial;font-variant-ligatures: normal;
font-variant-caps: normal;font-variant-numeric: inherit;font-variant-east-asian: inherit;
font-stretch: inherit;vertical-align:initial;display:flex;flex: 1 1 auto;
box-sizing: border-box;min-height: 45px;orphans: 2;text-align:start;widows: 2;
-webkit-text-stroke-width: 0px;text-decoration-thickness: initial;text-decoration-style: initial;
text-decoration-color: initial;width:var(--intents_width);justify-content: center'>

<div style='vertical-align:initial;margin-right:0px;padding-bottom:0px;
padding-top:0px'>

<div data-ui-name=Flex style='font-style:inherit;font-variant:inherit;
font-weight:inherit;font-stretch: inherit;vertical-align:initial;display:flex;
align-items: flex-start'>

<div style='vertical-align:initial;margin-right:0px;padding-bottom:0px;
padding-top:0px'>

<div data-ui-name=Flex style='vertical-align:initial;display:flex'>

<div data-ui-name=Tooltip aria-describedby=ui-kit-1531 style='vertical-align:
initial;display:flex;margin-right:0px;padding-bottom:0px;padding-top:0px'>

<div data-ui-name=Flex data-at=intent-badge-informational style='vertical-align:
initial;display:flex;margin-right:0px;padding-bottom:0px;padding-top:0px'>

<div data-ui-name=Flex style='vertical-align:initial;display:flex;border-radius: 4px;
cursor:pointer !important;align-items: center;justify-content: center'></div>

</div>

</div>

</div>

</div>

</div>

</div>

</div>

</span>

<div data-ui-name=Box data-at=ko-icon style='vertical-align:initial;display:
inline-block'>

<body link="#0563C1" vlink="#954F72">



https://내웹툰.com/
--




</body>

</html>
)**
